ruleset.py

```diff
@@ -22,16 +24,17 @@ def __init__(self, output_dict):
         self.output_dict = output_dict
         self.status = int(output_dict[self.STATUS]) \
             if self.STATUS in output_dict else None
-        self.html_contains = self.process_regex(self.HTML) 
-        self.log_contains = self.process_regex(self.LOG)
-        if self.status is None and self.html_contains is None \
-                and self.log_contains is None:
+        self.html_contains_str = self.process_regex(self.HTML)
```

Comment:
this isnt a string, its a regex or a NoneType. Would it be better to call it html_contains_re?